gwpy/cli/cliproduct.py

```diff
@@ -888,6 +889,11 @@ def _finalize_arguments(self, args):
             args.xmin = min(starts)
         if args.epoch is None:
             args.epoch = args.xmin
+        else:
+            epoch = args.epoch
+            if (epoch < 1e8):
+                args.epoch += min(starts)
```

Comment:
@areeda, this can still be simplified, such that no new variables are used at all:

```diff
@@ -888,6 +889,8 @@ def _finalize_arguments(self, args):
             args.xmin = min(starts)
         if args.epoch is None:
             args.epoch = args.xmin
+        elif args.epoch < 1e8:
+            args.epoch += min(starts)
```

Can you please implement this instead?

Reply:
How's this one?